pages/setting.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react'
```

Comment:
I think it should be named `settings.tsx` instead of `setting.tsx`.
The only reason I care is because this will be the URL end-point. 

Medium LOGAF.